docs/glossary/t/transfer-mode.lcdoc

```diff
@@ -12,6 +12,35 @@ Some transfer modes make the <object(glossary)> partly or completely
 transparent by combining each <pixel|pixel's> color with the color
 underneath it.
 
+The following is a non-exhaustive list of transfer modes:
+
+* <addMax(keyword)>
+* <adMin(keyword)>
+* <addPin(keyword)>
+* <addOver(keyword)>
+* <blend(keyword)>
+* <clear(keyword)>
+* <noOp(keyword)>
+* <notSrcAnd(keyword)>
+* <notSrcAndReverse(keyword)>
+* <notSrcBic(keyword)>
+* <notSrcCopy(keyword)>
+* <notSrcOr(keyword)>
+* <notSrcOrReverse(keyword)>
+* <notSrcXOr(keyword)>
+* <reverse(keyword)>
+* <set(keyword)>
+* <srcAnd(keyword)>
+* <srcAndReverse(keyword)>
+* <srcBic(keyword)>
+* <srcCopy(keyword)>
+* <srcOr(keyword)>
+* <srcOrReverse(keyword)>
+* <srcXOr(keyword)>
+* <subOver(keyword)>
+* <subPin(keyword)>
+* <transparent(keyword)>
+
 References: object (glossary), pixel (glossary)
```

Comment:
@seaniepie All of the transfer mode keywords need to be added to the `References:`. Also, in the bullet list you don't need to include the type `(keyword)` as the link specified in the references will be non-ambiguous! You only have to do that if there are two references with the same name of different type (as frequently occurs with `button (glossary)` and `button (object)` for example)